locations/spiders/maggianos_little_italy.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class MaggianosLittleItalySpider(scrapy.Spider):
+    name = "maggianos_little_italy"
+    item_attributes = {"brand": "Maggiano's Little Italy", "brand_wikidata": "Q6730149"}
+    allowed_domains = ["maggianos.com"]
+    start_urls = [
+        "https://www.maggianos.com/api//restaurant-data/",
+    ]
+
+    def parse_hours(self, hours):
+        opening_hours = OpeningHours()
+
+        for hour in hours:
+            day = hour["day_name"][:2]
+            opening_hours.add_range(day, open_time=hour["open_time"], close_time=hour["end_time"])
+
+        return opening_hours.as_opening_hours()
+
+    def parse(self, response):
+        data = response.json()
+
+        for place in data:
+            properties = {
+                "name": place["properties"]["business_name"],
+                "ref": place["properties"]["store_code"],
+                "addr_full": place["properties"]["slug"]["address_line_1"],
```

Comment:
```suggestion
                "street_address": place["properties"]["slug"]["address_line_1"],
```

Reply:
The source data also has field `address_line_2` so if this field is not blank, it should be appended to the street address:

```
"street_address": ", ".join(filter(None, [place["properties"]["slug"]["address_line_1"], place["properties"]["slug"]["address_line_2"]])),
```

Reply:
The source data also has a `full_address` field that can be extracted to `properties["addr_full"]`. Then you have both `properties["addr_full"]` and `properties["street_address"]`